onmt/bin/preprocess.py

```diff
@@ -125,19 +126,21 @@ def maybe_load_vocab(corpus_type, counters, opt):
     return src_vocab, tgt_vocab, existing_fields
 
 
-def build_save_dataset(corpus_type, fields, src_reader, tgt_reader, opt):
+def build_save_dataset(corpus_type, fields, r_src, r_tgt, r_align, opt):
```

Comment:
Why the change from X_reader to r_X? Especially when you're still using X_reader in other parts.

Reply:
It's just for shortening the line to not exceed 79 required by flake.